pumpkin-protocol/src/bytebuf/serializer.rs

```diff
@@ -121,7 +129,8 @@ impl<'a> ser::Serializer for &'a mut Serializer {
     where
         T: ?Sized + Serialize,
     {
-        unimplemented!()
+        self.output.put_var_int(&_variant_index.into());
+        _value.serialize(self)
```

Comment:
Can you may use value instead of _value ?,

Reply:
This is from the Serializer trait, I'm not sure I can change it

Reply:
Can you rename trait method parameters in rust? I've never tried

Reply:
Its so that clippy does not complain about dead code. You can just change the argument name

Reply:
> Can you rename trait method parameters in rust? I've never tried

yeah, Im pretty sure you can rename every argument in rust aslong it has the same type